scrapinghub/client/__init__.py

```diff
@@ -27,17 +27,23 @@ def request(self, *args, **kwargs):
 
 
 class ScrapinghubClient(object):
-    """Main class to work with Scrapinghub API.
+    """Main class to work with the Scrapy Cloud API.
 
-    :param auth: (optional) Scrapinghub APIKEY or other SH auth credentials.
-        If not provided, it will read, respectively, from 
+    :param auth: (optional) Scrapy Cloud API key or other Scrapy Cloud auth
+        credentials. If not provided, it will read, respectively, from
         ``SH_APIKEY`` or ``SHUB_JOBAUTH`` environment variables.
         ``SHUB_JOBAUTH`` is available by default in *Scrapy Cloud*, but it does
         not provide access to all endpoints (e.g. job scheduling), but it is allowed
         to access job data, collections, crawl frontier.
         If you need full access to *Scrapy Cloud* features, you'll need to
-        provide a Scrapinghub APIKEY through this argument or deploying ``SH_APIKEY``.
-    :param dash_endpoint: (optional) Scrapinghub Dash panel url.
+        provide a Scrapy Cloud API key through this argument or deploying
+        ``SH_APIKEY``.
+    :param dash_endpoint: (optional) Scrapy Cloud API URL.
+        If not provided, it will be read from the ``SHUB_APIURL`` environment
+        variable, or fall back to ``"https://app.zyte.com/api/"``.
+    :param endpoint: (optional) Scrapy Cloud storage API URL.
+        If not provided, it will be read from the ``SHUB_STORAGE`` environment
+        variable, or fall back to ``"https://storage.scrapinghub.com/"``.
```

Comment:
I'm not sure how to handle this but the `endpoint` parameter is not handled by the `ScrapinghubClient` directly, it's one of the additional arguments passed as `**kwargs` to the `HubstorageClient` class.